src/chapters/thymeleaf-views/studio.rst

```diff
@@ -0,0 +1,88 @@
+Studio: Spa Day!
+================
+
+After all of the hard work we have put into learning about Thymeleaf, it is time for a spa day!
+First, we need to put our knowledge of Thymeleaf to the test.
+Instead of heading out to our favorite spa, let's make an application for the owners!
+
+Our application needs to do the following:
+
+#. Display the user's name and skin type under their customer profile.
+#. Display the appropriate facial treatments for their skin type.
+#. Display the description of the spa's manicures or pedicures depending on the user's interest.
+
+As always, read through the whole page before starting to code.
+
+Setup
+-----
+
+Fork and clone the appropriate `repository <https://github.com/LaunchCodeEducation/spa-day-starter-code>`_. Check out the project via Version Control in IntelliJ.
+
+When you open the project and run it using ``bootRun``, go to ``localhost:8080`` and you should see a form!
+
+.. figure:: figures/startingform.png
+   :alt: Image showing an empty form with three fields for name, skin type, and whether the person wants a pedicure or manicure.
+
+The Customer Profile
+--------------------
+
+In ``controllers``, we have one controller called ``SpaDayController``. Inside ``SpaDayController``, we have three methods.
+
+#. The ``customerForm()`` method, which looks very similar to what we did in the last lesson.
```

Comment:
This is kinda picky, but can we list the methods here in the same order that they appear in the starter code?

Reply:
Dones! Thanks!